src/mondeath.cpp

```diff
@@ -405,7 +405,17 @@ void mdeath::worm( monster &z )
 
 void mdeath::disappear( monster &z )
 {
-    add_msg_if_player_sees( z, m_good, _( "The %s disappears." ), z.name() );
+    // The .compare() method returns 0 if equal
+    // Any other value is implicitly true
+    // so we need to reverse it
+    // substring(x,y) returns the y-character string starting at index x
+    bool begins_with_your = !( z.name().substring(0,4).compare("your") );
```

Comment:
`name()` returns a translated string, this won't work unless the language is english.

Reply:
Ah...shit. I can't off the top of my head think of a good fix for that.

Reply:
Is there an equivalent of `name()` - e.g., one used for debugging - that doesn't return a translated string?

Reply:
I think you should just have a field in JSON for monsters that enables this behavior, instead of trying to make guesses based on the name.